app/src/routes/login/components/login-form/login-form.vue

```diff
@@ -102,7 +102,7 @@ async function onSubmit() {
 <template>
 	<form novalidate @submit.prevent="onSubmit">
 		<v-input v-model="email" autofocus autocomplete="username" type="email" :placeholder="t('email')" />
-		<v-input v-model="password" type="password" autocomplete="current-password" :placeholder="t('password')" />
+		<interface-system-input-password v-model="password" autocomplete="current-password" />
```

Comment:
The two way `v-model` binding works by pure coincidence here. The interface expects the `value` prop. But since the `modelValue` prop is not given it is instead passed through as an attribute to the first root level component, the `v-input`, which happens to work with the `modelValue` prop. 

![Screenshot 2024-08-26 at 13 39 05](https://github.com/user-attachments/assets/6d2fb613-3566-4968-b3a5-95bce379c5f3)
![Screenshot 2024-08-26 at 13 38 56](https://github.com/user-attachments/assets/396425a8-ba5b-4596-b170-9349e23bcce4)

So the value binding here should be `:value="password" @input="password = $event"`